test/module/app/getGeneralSettings.test.js

```diff
@@ -3,106 +3,1221 @@
  * test app module
  */
 const nock = require('nock');
-
-const common = require('../../utils/common');
-
-const KintoneAPIException = require('../../../src/exception/KintoneAPIException');
-const Connection = require('../../../src/connection/Connection');
-const Auth = require('../../../src/authentication/Auth');
-const App = require('../../../src/module/app/App');
+const common = require('../utils/common');
+const {App, Auth, Connection, KintoneAPIException} = require('../../../src/main.js');
 
 const URI = 'https://' + common.DOMAIN;
+const preLiveRoute = '/k/v1/preview/app/settings.json';
```

Comment:
@vuong-le 
1. The const should be defined by underscore variable style
2. Using the clearly variable name
Ex: 
const APP_SETTINGS_ROUTE = '/k/v1/preview/app/settings.json';